lib/api/datatables.js

```diff
@@ -73,28 +73,10 @@ function collectionMembersDatatable(req, res) {
                     }
                 }
 
-                var memberTypeUrl = member.type?member.type:'http://wiki.synbiohub.org/wiki/Terms/SynBioHub#Unknown'
+                var memberTypeUrl = member.type.replace('http://sbols.org','http://sbolstandard.org')
                 var memberType = member.type?member.type.slice(member.type.lastIndexOf('#') + 1):'Unknown'
-
-                if (memberType === 'Collection') {
-                    memberTypeUrl = 'http://wiki.synbiohub.org/wiki/Terms/SynBioHub#Collection'
-                    memberType = 'Collection'
-                } else if (memberType === 'ComponentDefinition') {
-                    memberTypeUrl = 'http://wiki.synbiohub.org/wiki/Terms/SynBioHub#Part'
-                    memberType = 'Component'
-                } else if (memberType === 'ModuleDefinition') {
-                    memberTypeUrl = 'http://wiki.synbiohub.org/wiki/Terms/SynBioHub#Device'
-                    memberType = 'Module'
-                } else if (memberType === 'Sequence') {
-                    memberTypeUrl = 'http://wiki.synbiohub.org/wiki/Terms/SynBioHub#Sequence'
-                    memberType = 'Sequence'
-                } else if (memberType === 'Model') {
-                    memberTypeUrl = 'http://wiki.synbiohub.org/wiki/Terms/SynBioHub#Model'
-                    memberType = 'Model'
-                } else if (memberType === 'Attachment') {
-                    memberTypeUrl = 'http://wiki.synbiohub.org/wiki/Terms/SynBioHub#Attachment'
-                    memberType = 'Attachment'
-                }
+		if (memberType === 'ComponentDefinition') memberType = 'Component'
+		else if (memberType === 'ModuleDefinition') memberType = 'Module'
```

Comment:
Yes, since these are all now handled uniformly.  Namely, the member type is stripped off the end of the type URI, and the URL is at http://sbolstandard.org now, since that is where Goksel's ontology is.